docs/en_US/install/3ds.md

```diff
@@ -28,8 +39,46 @@ We make use of 3 Luma patches to connect your console to Pretendo:
 2) `0004013000003202` - Friends system module. This patch replaces the `https://nasc.nintendowifi.net` URL with our servers URL 
 3) `0004013000003802` - act system module. This patch replaces the `https://account.nintendo.net/v1/api/` URL with our servers URL
 
-## How does it work?
+<div class="tip">
+ℹ️ <b>Skip this step if Luma patches are already enabled on your console.</b>
+</div>
+
+In order to make use of the patches listed above, you will need to enable Luma patches on your console. Hold the SELECT button on your 3DS and power it on.
+
+You should be greeted to a black screen with a few tickboxes. You'll need to tick 'enable loading external FIRMS and modules' and 'enable game patching' to ensure Pretendo works correctly. After that, press START on your console to save and boot into your 3DS.
+
+## Installing Nimbus app to HOME Menu
+
+<div class="tip">
+ℹ️ <b>Skip this step if you downloaded the 3dsx only zip file.</b>
+</div>
+
+If you downloaded the combined or cia zip files, you can install the Nimbus application to the HOME Menu for quick and easy access. Here's how to do that.
+
+First of all, open up the FBI application on your 3DS HOME Menu. Once it is loaded, click on SD and then click on cias. Next, locate `nimbus.cia` and click on it. In the menu that appears, press the install and delete CIA button. This installs the application to your HOME Menu.
```

Comment:
FBI also ships as a 3dsx app, so it may not be on the home menu, and not everyone may have it already

```suggestion
Open FBI. If you do not have FBI, download the latest release from [GitHub](https://github.com/Steveice10/FBI/releases/latest). Select `SD`, then `cias`. Find and select `nimbus.cia`. Select either `Install CIA` or `Install and delete CIA`
```